frontend/src/scenes/insights/InsightNav/insightNavLogic.test.ts

```diff
@@ -0,0 +1,192 @@
+import { insightLogic } from 'scenes/insights/insightLogic'
+import { InsightLogicProps, InsightShortId, InsightType } from '~/types'
+import { insightNavLogic } from 'scenes/insights/InsightNav/insightNavLogic'
+import { expectLogic } from 'kea-test-utils'
+import { initKeaTests } from '~/test/init'
+import { MOCK_DEFAULT_TEAM } from 'lib/api.mock'
+import { useMocks } from '~/mocks/jest'
+
+describe('insightNavLogic', () => {
+    let theInsightLogic: ReturnType<typeof insightLogic.build>
+    let theInsightNavLogic: ReturnType<typeof insightNavLogic.build>
```

Comment:
going to steal this naming scheme for built insights

Reply:
I've just been using a naming scheme like `builtInsightLogic` for built logics 🙃. No strong opinions here.